grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -341,30 +342,32 @@ class StockMovementApiController {
     }
 
     def exportPickListItems() {
```

Comment:
both documents are nearly identical and I wasn't sure what would be the better route, create a separate endpoint for both or parametrize the one existing endpoint.
The difference is that **lot**, **binLocation** and **expirationDate** are empty on the template and quantity on the template is `RequsitionItem.quantity`.

I chose to add a boolean parameter `template` for the same endpoint but I am open to your criticism if this approach is not a correct one.

Reply:
I was really confused by this. I figured the template would be a completely blank template, but I went back to the ticket and now I'm understanding it a bit more. I don't think we want to pass a boolean parameter for this. This should be something derived - each line knows whether it has picklist items associated with it or not.